packages/zowe-explorer/src/trees/job/JobActions.ts

```diff
@@ -253,6 +253,17 @@ export class JobActions {
         statusMsg.dispose();
     }
 
+    public static async loadMoreRecords(doc: vscode.TextDocument): Promise<void> {
+        if (!doc) {
+            vscode.window.showErrorMessage("No document found.");
```

Comment:
Please add localization to this string:

```suggestion
            Gui.errorMessage(vscode.l10n.t("No document found."));
```